runtime/nls/dump/j9dmp.nls

```diff
@@ -506,3 +506,11 @@ J9NLS_DMP_INVALID_OR_RESERVED.user_response=select a different value.
 J9NLS_DMP_INVALID_OR_RESERVED.sample_input_1=suspendwith
 J9NLS_DMP_INVALID_OR_RESERVED.link=
 # END NON-TRANSLATABLE
+
+J9NLS_DMP_TOO_MANY_DUMP_OPTIONS=Too many dump options specified
```

Comment:
Please add the `MAX_DUMP_OPTS` value to this message so users will know how many options they can have

Reply:
That's a bit misleading as approximately 20 slots are taking by the VM for default settings.  I could capture the latter value just before the while loop and subtract it from `MAX_DUMP_OPTS`.